langchain4j/src/main/java/dev/langchain4j/agent/tool/ToolParameterHandler.java

```diff
@@ -0,0 +1,17 @@
+package dev.langchain4j.agent.tool;
+
+import java.util.Objects;
+import java.util.function.Function;
+
+@FunctionalInterface
+public interface ToolParameterHandler<T, R> extends Function<T, R> {
```

Comment:
please make this and all other classes that should not be exposed to the user of the library package-private

Reply:
`ToolDoubleParameterHandlers` & `ToolParameterHandler` has done. see [c33f70d](https://github.com/langchain4j/langchain4j/pull/136/commits/c33f70d65138aea401ac55f54833428272dfc773)